tests/lib/muinstaller/main.go

```diff
@@ -421,6 +418,83 @@ func detectStorageEncryption(seedLabel string) (bool, error) {
 	return details.StorageEncryption.Support == client.StorageEncryptionSupportAvailable, nil
 }
 
+func fillPartiallyDefinedVolume(vol *gadget.Volume, bootDevice string) error {
+	if len(vol.Partial) == 0 {
+		return nil
+	}
+
+	logger.Noticef("partial gadget for: %q", vol.Partial)
+
+	// TODO: partial structure, as it is not clear what will be possible when set
```

Comment:
aren't we testing partial structure at least in so far that there is an extra unknown structure? this TODO maybe needs to be a bit more precise

Reply:
Right, I wouldn't say it is a TODO anymore. I've removed this and instead added a function comment on the things that we fill.